WooCommerce/Classes/POS/Presentation/CardReaderConnection/CardReaderConnectionStatusView.swift

```diff
@@ -16,18 +16,29 @@ struct CardReaderConnectionStatusView: View {
         Group {
             switch connectionViewModel.connectionStatus {
                 case .connected:
-                    Text("Reader connected")
+                    HStack(spacing: Layout.buttonImageAndTextSpacing) {
+                        Image(systemName: "wave.3.forward.circle")
```

Comment:
For some future version we can use the "pallete" setup for this icon:

<img width="474" alt="Screenshot 2024-06-07 at 10 11 40" src="https://github.com/woocommerce/woocommerce-ios/assets/6242034/c2217a8e-a3be-440e-87a4-d830d5d885d0">

And even animate it:

<img width="472" alt="Screenshot 2024-06-07 at 10 12 51" src="https://github.com/woocommerce/woocommerce-ios/assets/6242034/58e04bb1-5b9b-4443-adc5-e1e463389a75">

Reply:
Yup it's nice having these color customizations from SF Symbols.